landoapi/validation.py

```diff
@@ -6,6 +6,7 @@
 from connexion import ProblemException
 
 REVISION_ID_RE = re.compile(r"^D(?P<id>[1-9][0-9]*)$")
+VALID_EMAIL_RE = re.compile(r"[^@ \t\r\n]+@[^@ \t\r\n]+\.[^@ \t\r\n]+")
```

Comment:
This may be a misnomer since it could catch invalid email addresses too. Maybe `ACCEPTED_EMAIL_RE` is a better name. There probably is a more suitable regex that is available on the internet.

An example of an invalid email that will pass the above: `-@...`